File: RemoveRecursionRefactoring/src/ro/pub/cs/diploma/inspections/ReplaceDeclarationsHavingInitializersWithAssignmentsInspection.java
package ro.pub.cs.diploma.inspections;

public class ReplaceDeclarationsHavingInitializersWithAssignmentsInspection extends DummyInspection {
  @Override
  protected String getKey() {
    return "replace.declarations.having.initializers.with.assignments";
  }

  @Override
  protected int getSteps() {
    return 8;
  }
}

File: RemoveRecursionRefactoring/src/ro/pub/cs/diploma/inspections/ReplaceDeclarationsHavingInitializersWithAssignmentsInspection.kt
package ro.pub.cs.diploma.inspections

class ReplaceDeclarationsHavingInitializersWithAssignmentsInspection : DummyInspection() {
  override val key
    get() = "replace.declarations.having.initializers.with.assignments"

  override val steps
    get() = 8
}
